src/sentry/static/sentry/app/views/organizationDiscover/discover.jsx

```diff
@@ -239,164 +249,110 @@ export default class OrganizationDiscover extends React.Component {
 
     return (
       <Discover>
-        <DiscoverBody>
-          <Sidebar w={320}>
-            <DiscoverHeader>{t('Discover')}</DiscoverHeader>
-            <SidebarHeader align="center" px={space(4)} py={space(1.5)}>
-              <Box flex="1">
-                <DiscoverSubHeader>{t('Query')}</DiscoverSubHeader>
-              </Box>
-              <Box>
-                <Button
-                  size="xsmall"
-                  onClick={this.reset}
-                  style={{marginRight: space(1)}}
-                >
-                  {t('Reset')}
-                </Button>
-                <Button
-                  size="xsmall"
-                  onClick={this.runQuery}
-                  priority="primary"
-                  busy={isFetchingQuery}
-                >
-                  {t('Run')}
-                  {isFetchingQuery && <ButtonSpinner />}
-                </Button>
-              </Box>
-            </SidebarHeader>
-            <Fieldset>
-              <SidebarLabel htmlFor="fields" className="control-label">
-                {t('Summarize')}
-              </SidebarLabel>
-              <SelectControl
-                name="fields"
-                multiple={true}
-                placeholder={this.getSummarizePlaceholder()}
-                options={fieldOptions}
-                value={currentQuery.fields}
-                onChange={val => this.updateField('fields', val.map(({value}) => value))}
-                clearable={true}
+        <Sidebar>
+          <PageTitle>{t('Discover')}</PageTitle>
+          <SidebarHeader>
+            <SidebarTitle>{t('Query')}</SidebarTitle>
+            <Box>
+              <Button size="xsmall" onClick={this.reset} style={{marginRight: space(1)}}>
```

Comment:
Can we make a styled component for this?

Reply:
I usually just stick stuff into a `<Box mr={1}>`

Reply:
I think that's slightly better, but trying to have us avoid these cryptic presentational props. I think `direction`, `justify`, etc is more ok since it's a bit more semantic.